graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/functions/messages/DropMessage.java

```diff
@@ -53,6 +54,10 @@ public FunctionDescriptor<Void> descriptor() {
                         messageParam
                 ))
                 .description("Discards a message from further processing, after completing the current stage")
+                .ruleBuilderEnabled()
+                .ruleBuilderName("Drop message")
+                .ruleBuilderTitle("Remove message from the processing pipeline after rule")
```

Comment:
Prefer "after current stage", since there is often confusion as to when exactly the message gets dropped.